ess/src/API/EMBC.ESS/Managers/Teams/TeamsManager.cs

```diff
@@ -238,9 +238,8 @@ public async Task<string> Handle(ClaimSupplierCommand cmd)
                 SupplierId = cmd.SupplierId,
             })).Items.SingleOrDefault(m => m.Id == cmd.SupplierId);
             if (supplier == null) throw new NotFoundException($"Supplier {cmd.SupplierId} not found", cmd.SupplierId);
-            if (supplier.Team != null && supplier.Team.Id != null) throw new BusinessLogicException($"Supplier already has a primary team");
 
-            supplier.Team = new Resources.Suppliers.Team { Id = cmd.TeamId };
+            supplier.PrimaryTeams = supplier.PrimaryTeams.Concat(new[] { new Resources.Suppliers.Team { Id = cmd.TeamId } });
```

Comment:
Append?

Reply:
complains IEnumerable<Team> does not contain a definition for Append.